src/pybind/mgr/prometheus/module.py

```diff
@@ -1267,13 +1266,20 @@ def _get_pool_info(pool: Dict[str, Any]) -> Tuple[str, str]:
             if 'options' in pool:
                 compression_mode = pool['options'].get('compression_mode', 'none')
 
+            application_metadata_str = ''
+            if not(pool['pool_name'].startswith('.')):
+                application_metadata = pool.get('application_metadata', {})
+                application_metadata_str = ', '.join(f'{k}' for k, v in application_metadata.items())
```

Comment:
```suggestion
                application_metadata_str = ', '.join(str(k) for k in application_metadata)
```
NIT, since we're not using the values.